Wikipedia/Code/PageHistoryFilterCountCollectionViewCell.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+class PageHistoryFilterCountCollectionViewCell: UICollectionViewCell {
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var imageLabel: UILabel!
+
+    @IBOutlet private weak var rightSeparator: UIView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        updateFonts()
```

Comment:
Due to the [traitCollection changes in iOS 13 ](https://developer.apple.com/documentation/ios_ipados_release_notes/ios_13_release_notes) this initial `updateFonts` might be off. I wouldn't worry about it if we're not seeing any issues though